integration/heapster_api_test.go

```diff
@@ -42,16 +44,18 @@ const (
 )
 
 var (
-	testZone               = flag.String("test_zone", "us-central1-b", "GCE zone where the test will be executed")
-	kubeVersions           = flag.String("kube_versions", "", "Comma separated list of kube versions to test against. By default will run the test against an existing cluster")
-	heapsterControllerFile = flag.String("heapster_controller", "../deploy/kube-config/standalone-test/heapster-controller.yaml", "Path to heapster replication controller file.")
-	heapsterServiceFile    = flag.String("heapster_service", "../deploy/kube-config/standalone-test/heapster-service.yaml", "Path to heapster service file.")
-	heapsterImage          = flag.String("heapster_image", "heapster:e2e_test", "heapster docker image that needs to be tested.")
-	avoidBuild             = flag.Bool("nobuild", false, "When true, a new heapster docker image will not be created and pushed to test cluster nodes.")
-	namespace              = flag.String("namespace", "heapster-e2e-tests", "namespace to be used for testing, it will be deleted at the beginning of the test if exists")
-	maxRetries             = flag.Int("retries", 20, "Number of attempts before failing this test.")
-	runForever             = flag.Bool("run_forever", false, "If true, the tests are run in a loop forever.")
-	testUser               = flag.String("test_user", "", "GCE user when copy file to host")
+	testZone                   = flag.String("test_zone", "us-central1-b", "GCE zone where the test will be executed")
+	kubeVersions               = flag.String("kube_versions", "", "Comma separated list of kube versions to test against. By default will run the test against an existing cluster")
+	heapsterControllerFile     = flag.String("heapster_controller", "../deploy/kube-config/standalone-test/heapster-controller.yaml", "Path to heapster replication controller file.")
+	heapsterServiceFile        = flag.String("heapster_service", "../deploy/kube-config/standalone-test/heapster-service.yaml", "Path to heapster service file.")
+	heapsterRBACFile           = flag.String("heapster_rbac", "../deploy/kube-config/rbac/heapster-rbac.yaml", "Path to heapster rbac file.")
```

Comment:
Please use different rbac file, for your testing service account. The current one binds cluster role system:heapster to the service account from kube-system namespace.

Reply:
Added a new heapster-rbac.yaml file to standalone-test dir.